sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/TraceHelperNewTests.cs

```diff
@@ -90,24 +90,26 @@ public void GetNewOperationName_WithNullHttpMethod_ReturnsActivityDisplayName(st
         }
 
         [Fact]
-        public void GetNewOperationName_WithNullHttpRoute_ReturnsActivityDisplayName()
+        public void GetNewOperationName_WithNullHttpRoute_ReturnsUrlWithVerb()
         {
             // Arrange
-            using var tracerProvider = Sdk.CreateTracerProviderBuilder().AddSource(nameof(GetNewOperationName_WithNullHttpRoute_ReturnsActivityDisplayName)).Build();
-            using var activitySource = new ActivitySource(nameof(GetNewOperationName_WithNullHttpRoute_ReturnsActivityDisplayName));
+            using var tracerProvider = Sdk.CreateTracerProviderBuilder().AddSource(nameof(GetNewOperationName_WithNullHttpRoute_ReturnsUrlWithVerb)).Build();
+            using var activitySource = new ActivitySource(nameof(GetNewOperationName_WithNullHttpRoute_ReturnsUrlWithVerb));
             using var activity = activitySource.StartActivity(
                 ActivityName,
                 ActivityKind.Server);
             activity?.Stop();
 
             var tagObjects = AzMonList.Initialize();
             AzMonList.Add(ref tagObjects, new KeyValuePair<string, object?>(SemanticConventions.AttributeHttpRequestMethod, "POST"));
+            AzMonList.Add(ref tagObjects, new KeyValuePair<string, object?>(SemanticConventions.AttributeServerAddress, "example.com"));
```

Comment:
`AttributeServerAddress` is a required field, we will not return activity display name.